src/test/java/io/xconn/cryptobox/SecretBoxTest.java

```diff
@@ -0,0 +1,85 @@
+package io.xconn.cryptobox;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import static io.xconn.cryptobox.SecretBox.MAC_SIZE;
```

Comment:
we should order the imports
1. standard library
2. third-party
3. project code